enhancements/network/no_overlay_mode.md

```diff
@@ -0,0 +1,965 @@
+
+## Graduation Criteria
+
+<!-- **Note:** *Section not required until targeted at a release.* -->
+
+<!-- Define graduation milestones. -->
+
+<!-- These may be defined in terms of API maturity, or as something else. Initial proposal -->
+<!-- should keep this high-level with a focus on what signals will be looked at to -->
+<!-- determine graduation. -->
+
+<!-- Consider the following in developing the graduation criteria for this -->
+<!-- enhancement: -->
+
+<!-- - Maturity levels -->
+<!--   - [`alpha`, `beta`, `stable` in upstream Kubernetes][maturity-levels] -->
+<!--   - `Dev Preview`, `Tech Preview`, `GA` in OpenShift -->
+<!-- - [Deprecation policy][deprecation-policy] -->
+
+<!-- Clearly define what graduation means by either linking to the [API doc definition](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning), -->
+<!-- or by redefining what graduation means. -->
+
+<!-- In general, we try to use the same stages (alpha, beta, GA), regardless how the functionality is accessed. -->
+
+<!-- [maturity-levels]: https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions -->
+<!-- [deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/ -->
+
+<!-- **If this is a user facing change requiring new or updated documentation in [openshift-docs](https://github.com/openshift/openshift-docs/), -->
+<!-- please be sure to include in the graduation criteria.** -->
+
+<!-- **Examples**: These are generalized examples to consider, in addition -->
+<!-- to the aforementioned [maturity levels][maturity-levels]. -->
+
+We are going to develop this feature in two phases:
+- phase 1: we will support no-overlay mode in the unmanaged configuration: an FRRConfig and a RouteAdvertisements to configure the BGP topology are provided by the cluster administrator.
+- phase 2: we will add support for the managed configuration: the FRRConfig and RouteAdvertisements are generated by OVN-Kubernetes based on the new API parameters.
+
+Until phase 2 is complete, the feature will be considered in Tech Preview and will be enabled by a feature gate.
+
+An extra goal, not required for GA, will be to add support for the managed network topology `routeReflector`. In this managed topology, every node labeled with `k8s.ovn.org/internal-bgp-role=route-reflector` will run a BGP route reflector; all other nodes will only peer with the route reflectors. This is particularly useful for large clusters, as it reduces the total number of BGP connections compared to a full-mesh topology. Given that it requires to first add native support for route reflectors in FRR-Kubernetes and, in the context of no-overlay mode, it is an optimization rather than a requirement, we will consider it as a future goal after GA.
```

Comment:
I wouldn't even mention this, a t this on this level of detail. It is likely going to need its own enhancement with a potential different approach that what's described here.
nit: FRR-k8s

Reply:
You're right. I wrote this before we removed it entirely from the upstream enhancement.